aspnet/hosting/servicing.rst

```diff
@@ -0,0 +1,34 @@
+.. _hosting-servicing:
+
+Servicing
+=========
+
+By `Sourabh Shirhatti`
+
+ASP.NET 5 supports servicing of runtime components (DNX) and packages through Windows Update Services which will deliver updates to patch any vulnerabilities if and when they are discovered. This document provides an overview of how to setup your Windows Server correctly to receive updates.
+
+Breadcrumbs Directory
+---------------------
+
+All serviceable assemblies will leave a breadcrumb in the ``BreadcrumbStore`` Directory. At the time of servicing Windows Update looks in this directory to figure out which assemblies are used on the server and need patching. This directory must be correctly protected by ACLs to prevent rogue applications from deleting entries from this directory. Run the powershell snippet below in an elevated prompt to create the ``BreadcrumbStore`` directory and ACL it correctly.
+
+
```

Comment:
Link to the script in github rather than copy it - that way if it changes then it doesn't need updating here.